client/src/components/SignIn.tsx

```diff
@@ -193,10 +193,11 @@ function SignIn(props: { handleChange: Function; showAlert: Function }) {
           error={signinFormErrors.password !== ''}
           helperText={signinFormErrors.password}
         />
-        <FormControlLabel
-          control={<Checkbox value='remember' color='primary' />}
-          label='Remember me'
-        />
+        {/* will be implemented in future PR (after search) */}
+        {/* <FormControlLabel
+          control={<Checkbox value="remember" color="primary" />}
+          label="Remember me"
+        /> */}
```

Comment:
```suggestion
```

Reply:
I was worried about keeping this in, since, if we don't end up implementing this functionality, yet we have this button, we don't want them asking about it or to show them lol